amplipi/app.py

```diff
@@ -79,6 +81,7 @@
 
 app.add_exception_handler(NotAuthenticatedException, not_authenticated_exception_handler)
 
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
could you give this logger a name, and import it in each of the files?

I was imagining something more along the lines of [this](https://docs.python.org/3/howto/logging.html#configuring-logging). This is actually what gives us the flexibility to move log lines around, increase logging on particular components as we need, direct certain things to files. You don't need to bother with any of the formatting or stream handler stuff - just creating a particular `logger` instance and getting it in each module is plenty.